scripts/apps/web-publisher/controllers/WebPublisherManagerController.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * @ngdoc controller
+ * @module superdesk.apps.web_publisher
+ * @name WebPublisherManagerController
+ * @requires api/superdesk.apps.web_publisher/service/PublisherFactory publisher 
+ * @requires superdesk.core.services.modal modal
```

Comment:
```js
@requires modal
```
is enough here, since that is the name of the service. It is requiring a service, not a module.